### PR TITLE
[DK-847] Feature/python logger

### PR DESCRIPTION
**Pull request by nguyenqh-visenze**

### CODE REVIEW

**Review by hongshibao — APPROVED** [review]

File: python/recordio/recordio.py [hunk 1]
```diff
@@ -22,12 +22,18 @@ class writer(object):
     writer is a recordio writer.
     """
 
-    def __init__(self, path, maxChunkSize = -1, compressor = -1):
+    def __init__(self, path, maxChunkSize=-1, compressor=-1, logger=None):
```
Comment (hongshibao) [inline python/recordio/recordio.py]: Can you help to change `maxChunkSize` to `max_chunk_size` ?

File: python/setup.py [hunk 2]
```diff
@@ -6,7 +6,7 @@ def has_ext_modules(foo):
 
 setup(
     name='visenze_recordio',
-    version='0.1',
+    version='0.2',
```
Comment (ghost) [inline python/setup.py:9]: Where does this version number come from? I checked [gemfury](https://manage.fury.io/dashboard/visenze/package/pkg_21uYbX/version/ver_Ux1VK) and it was still 0.1.

File: python/recordio/recordio.py (hunk 1 above)
Comment (ghost) [reply python/recordio/recordio.py]: Another question: is this change only for debug? Or is there any other background that necessitates this change?

**Review by ghost — APPROVED** [review]
@nguyenqh-visenze Could you address the version number of `visenze_recordio`?

File: python/recordio/recordio.py (hunk 1 above)
Comment (nguyenqh-visenze) [reply python/recordio/recordio.py]: yeah, only to add more log to facilitate debugging

File: python/setup.py (hunk 2 above)
Comment (nguyenqh-visenze) [reply python/setup.py:9]: after this one is merged then it'll be released to gemfury as 0.2

Comment (ghost) [reply python/setup.py:9]: I see.

**Pull request merged**